src/js/view/controls/next-display-icon.js

```diff
@@ -1,11 +1,16 @@
 import displayIconTemplate from 'view/controls/templates/display-icon';
 import NEXT_ICON from 'assets/SVG/next.svg';
 import utils from 'utils/helpers';
+import UI from 'utils/ui';
 
 export default class NextDisplayIcon {
-    constructor(model) {
+    constructor(model, api) {
         const element = utils.createElement(displayIconTemplate('next', model.get('localization').next, NEXT_ICON));
 
+        new UI(element).on('click tap', function() {
+            api.next();
```

Comment:
@egreaves there is a playlistNext() api call that takes in a meta, https://github.com/jwplayer/jwplayer/blob/master/src/js/api/api.js#L748 ; next does not take one
the difference between the 2 is at the last item of a playlist, playlistNext will set item1, whereas next will start playing related